.circleci/config.yml

```diff
@@ -207,6 +207,44 @@ jobs:
                     aws s3 sync --only-show-errors . s3://cardigan.wellcomecollection.org
                 popd
             fi
+  edge_lambdas_build:
+    docker:
+      - image: circleci/node:8.11.3
+    working_directory: /home/circleci/wellcomecollection.org/router/webapp
+    steps:
+      - add_ssh_keys
+      - restore_cache:
+          key: repo_{{ .Environment.CIRCLE_SHA1 }}
+      - run:
+          name: Install deps
+          command: yarn
+      - run:
+          name: Run tests
+          command: yarn test
+      - run:
+          name: Build lambda ZIP
+          command: |
+            zip -r edge_lambda_origin.zip ab_testing.js origin.js redirector.js redirects.js wiRedirector.js
+            mkdir -p /tmp/artifacts
+            cp edge_lambda_origin.zip /tmp/artifacts
```

Comment:
Lots of hard coding of the lambda name and bucket, not sure how to share config with circle / terraform.
[Possibly using terraform to deploy the lambda](https://www.terraform.io/docs/providers/aws/r/s3_bucket_object.html) and storing the state?